ckanext/unaids/plugin.py

```diff
@@ -99,3 +100,32 @@ def get_validators(self):
         return{
             'if_empty_guess_format': if_empty_guess_format
         }
+
+
+class UNAIDSReclineView(ReclineViewBase):
+    '''
+    This override of the recline view plugin allows data explorers to be auto
+    created for geojson files.
+    '''
+
+    def info(self):
+        return {'name': 'unaids_recline_view',
```

Comment:
We have to change this line so the view is uniquely identified and uncoupled fro the original recline view.